src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double area();
```

Comment:
```suggestion
    double getArea();
```

https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names

Reply:
Fixed